### PR TITLE
Remove deadcode.

### PR DESCRIPTION
**Pull request by rabbah**
Closes https://github.com/apache/incubator-openwhisk/issues/3861.

## Related issue and scope

- [x] I opened an issue to propose and discuss this change (#3861)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [x] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [x] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [ ] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

File: tests/src/test/scala/common/WskCliOperations.scala [hunk 1]
```diff
@@ -972,7 +972,7 @@ class CliGatewayOperations(val wsk: RunCliCmd) extends GatewayOperations {
 
 object Wsk {
   val binaryName = "wsk"
-  val defaultCliPath = if (WhiskProperties.useCLIDownload) getDownloadedGoCLIPath else WhiskProperties.getCLIPath
+  val defaultCliPath = WhiskProperties.getCLIPath
```
Comment (dubee) [inline tests/src/test/scala/common/WskCliOperations.scala]: Is the definition below for `getDownloadedGoCLIPath` still needed?

**Comment by rabbah** [comment]
@mdeuser can you please take care of this one - if we don't need this code anywhere downstream let's remove it.

**Comment by rabbah** [comment]
The code for "use.cli.download" should be redone - the path is assumed to be some arbitrary path really, might as well put it where the test harness expects it in openwhisk/bin. As such, since there is no reference in this project otherwise to the code, removing it makes sense to me.

**Comment by mdeuser** [comment]
pg2 4208

**Comment by rabbah** [comment]
An alternative to removing this boolean flag and the arbitrary location for the binary, is to have the ansible parameter point to a specific location instead. I'm not doing this since I see no real value in it or having the flag until someone tells me otherwise.

**Comment by rabbah** [comment]
Thanks @dubee for the update. If we need to preserve the behavior I prefer we do it in an alternate way that's cleaner IMO (the param should point to the binary location).

**Pull request merged**